Source/WebCore/platform/graphics/GraphicsTypesGL.h

```diff
@@ -47,6 +47,7 @@ typedef float GCGLclampf;
 typedef char GCGLchar;
 typedef void* GCGLsync;
 typedef void GCGLvoid;
+typedef void* GCGLimage;
```

Comment:
This might be part of the learning thing, you might mark this up as GCEGLimage.

Reply:
Yes.